README.md

```diff
@@ -2,5 +2,7 @@
 
 [![Sugaroid Bot](https://img.shields.io/badge/sugaroid-discord-%23f70049)](https://github.com/sugaroidbot/sugaroid)
 
+Discord wrapper for the Sugaroid Bot.
+
+![Sugaroid Discord Bot at work](https://g.sugaroid.srevinsaju.me/docs/_images/sugaroid_discord.png)
 
-![Sugaroid Discord Bot at work](http://g.sugaroid.srevinsaju.me/docs/img/sugaroid_discord.png)
```

Comment:
Why do you want to force use a non secure link. i.e `http`?

Reply:
@srevinsaju I guess you've mixed up with my additions and deletions

Reply:
```
Discord wrapper for the Sugaroid Bot.

![Sugaroid Discord Bot at work](https://g.sugaroid.srevinsaju.me/docs/_images/sugaroid_discord.png)
```

This is what I've added